Tests/MapboxCoreNavigationTests/NavigationEventsManagerTests.swift

```diff
@@ -78,7 +78,7 @@ class NavigationEventsManagerTests: XCTestCase {
         ])
         let eventTimeout = 0.3
         let route = Fixture.route(from: "DCA-Arboretum", options: routeOptions)
-        let dataSource = MapboxNavigationService(route: route, routeIndex: 0, routeOptions: routeOptions)
+        let dataSource = MapboxNavigationService(route: route, routeIndex: 0, routeOptions: routeOptions, directions: directions, simulating: .never)
```

Comment:
The default value for `simulating` is `onPoorGPS`; would setting it to `never` affect the test behavior?

Reply:
Thanks for the suggestion, I'll fix this part